elements/rh-dialog/docs/40-accessibility.md

```diff
@@ -63,8 +63,24 @@ Only the close button and any interactive elements are selectable.
 
 ### Backdrop
 
-A dialog will not close by users clicking or tapping the backdrop or outside of the container.
+A dialog will close by users clicking or tapping the backdrop or outside of the container.
+
+
+## Accessible labels
+
+Each dialog needs an accessible name. If a dialog has a heading tag in the `header` or default slot, this component will automatically apply an appropriate ID to the heading tag and an `aria-labelledby` attribute to the `<dialog>`.
+
+Users can optionally provide an `accessible-label` attribute which overrides the built-in `aria-labelledby` functionality:
+
+```html
+<rh-dialog accessible-label="Page Properties" trigger="first-modal-trigger">
+  ...
+</rh-dialog>
+```
+
+The `accessible-label` attribute assigns an `aria-label` to the `<dialog>` element.
```

Comment:
```suggestion
```

i don't think this information (the "how") is relevant to consumers. Rather, the relevant part is that applying `accessible-label` to rh-dialog makes the dialog more accessible to sr users.

same for the comment about `aria-labelledby` above. 

we are in the business of making accessible html elements with custom, public apis. the exact, granular details of what happens in the shadow root and in memory (ElementInternals) should not be of any concern to consumers of the design system.

Reply:
Resolved in [303f85f](https://github.com/RedHat-UX/red-hat-design-system/pull/2078/commits/303f85f107a053833d07585d07cff8bdd74f2cff).